config/initializers/devise.rb

```diff
@@ -19,7 +19,7 @@
   env["omniauth.strategy"].options[:client_id] = SiteConfig.apple_client_id
   env["omniauth.strategy"].options[:scope] = "email name"
   env["omniauth.strategy"].options[:key_id] = SiteConfig.apple_key_id
-  env["omniauth.strategy"].options[:pem] = SiteConfig.apple_pem
+  env["omniauth.strategy"].options[:pem] = SiteConfig.apple_pem.to_s.gsub("\\n", "\n")
```

Comment:
The backslashes are escaped when stored in SiteConfig. This fixes that problem and makes it possible to add the private key via the Admin page